server/svix-server/src/v1/endpoints/endpoint/crud.rs

```diff
@@ -58,9 +59,14 @@ pub(super) async fn list_endpoints(
     })
     .collect();
 
-    Ok(Json(EndpointOut::list_response_no_prev(
+    if is_prev {
```

Comment:
I saw the slack thread, so I know this is how we typically correct the ordering. But as you saw, it is a bit brittle since it's _very_ easy to forget.

I wonder if we could add this logic to `::list_response`. Just `reverse` in `list_response_inner`, so we're less likely to forget in the future.

Reply:
I fully agree with this, and in fact, we've done similar things elsewhere. This is extremely brittle (as you said), and extremely easy to miss.

Reply:
This can't be generically moved into `list_response` as is because there are other listing endpoints that use it that do ordering the opposite way by default (highest to lowest). At the very least we'd need to add an extra parameter to it to tell what the data looks like and/or add another method like `list_response_ascending` or something like that.

Reply:
Yeah, split to two functions then, list_response_asceding and descending. It sounds reasonable, and makes it less error prone.

Reply:
How does https://github.com/svix/svix-webhooks/pull/812/files#diff-9cb85d646fc32039b2a4538aae5cabf2da93cdd2ad85f826479937ac88206f09L231-R269 look?